Automattic/sensei
# Export list of lessons in course export file and remove course from lessons

### Changes proposed in this Pull Request

* Puts the lesson association and ordering in the course export file under `Lessons`. 
* Removes the course field from the lesson export (keeps it in schema until we remove it from importer).
* Next step will be updating the importer to accept this new schema.

### Testing instructions

* Create multiple courses with multiple lessons.
* Rearrange the order of the lessons.
* Run the exporter and make sure the ID list for `Lessons` shows all the lesson IDs in the order that they are set.
